std_msgs/Header header

# Network RSSI
int8 net_RSSI_dBm
int8 net_percent

# CPU Temp
int8 temp_package
int8 temp_core_0
int8 temp_core_1
int8 temp_core_2
int8 temp_core_3
int8 temp_core_4
int8 temp_core_5

# CPU Percent
int8 cpu_percent
int8 percent_0
int8 percent_1
int8 percent_2
int8 percent_3
int8 percent_4
int8 percent_5
